packages/core/src/routes/admin-user-role.ts

```diff
@@ -99,6 +99,44 @@ export default function adminUserRoleRoutes<T extends AuthedRouter>(
     }
   );
 
+  router.patch(
```

Comment:
This should've been a `put` method request, as you are replacing the roles, not merging it

Reply:
Yes, that makes sense. Just had it resolved.